.eslintrc

```diff
@@ -25,5 +25,17 @@
   "rules": {
     "valid-jsdoc": [0],
     "require-jsdoc": [0],
+    "quotes": ["error", "backtick"],
+    "jsx-quotes": ["error", "prefer-double"],
+    "max-len": ["error", 80],
+    "semi": ["error", "never"],
+    "object-curly-spacing": ["error", "always"],
+    "comma-dangle": ["error", {
```

Comment:
PS: And if you use an IDE that has a prettier-eslint extension / plugin, I encourage you to try it out with format on save set to true. Makes me smile every time 🙂